newton/core/model.py

```diff
@@ -114,12 +115,6 @@ def __init__(self, device=None):
         """Collision radius of each shape used for bounding sphere broadphase collision checking, shape [shape_count], float."""
         self.shape_contact_pairs = None
```

Comment:
Do we still need these `shape_contact_pairs` in the Model?

Reply:
I'm not sure if it makes sense to have this creation of shape pairs also as a method in the `CollisionPipeline`? This will implement the current behavior where collision groups and pair-wise filters are "hard-coded", but in the future we want to allow collision groups and filters to be handled at runtime. We could also tackle this in a later PR once the geometry module is getting built out more.